src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/seed/SeedCas1ApAreaEmailAddressTest.kt

```diff
@@ -96,9 +96,6 @@ class SeedCas1ApAreaEmailAddressTest : SeedTestBase() {
     )
 
     seedService.seedData(SeedFileType.approvedPremisesApAreaEmailAddresses, "valid-csv.csv")
-
-    assertThat(apAreaRepository.findByIdentifier("SWSC")!!.emailAddress).isEqualTo("swsc@test.com")
-    assertThat(apAreaRepository.findByIdentifier("Mids")!!.emailAddress).isEqualTo("mids@midlands.com")
```

Comment:
as mentioned in another comment, we can completely remove this test and seed job

Reply:
This test has been removed too.